README.rst

```diff
@@ -1,6 +1,30 @@
 PennyLane-Lightning Plugin
 ##########################
 
+.. image:: https://img.shields.io/github/workflow/status/PennyLaneAI/pennylane-lightning/Testing/master?logo=github&style=flat-square
+    :alt: GitHub Workflow Status (branch)
+    :target: https://github.com/PennyLaneAI/pennylane-lightning/actions?query=workflow%3ATesting
+
+.. image:: https://img.shields.io/codecov/c/github/PennyLaneAI/pennylane-lightning/master.svg?logo=codecov&style=flat-square
+    :alt: Codecov coverage
+    :target: https://codecov.io/gh/PennyLaneAI/pennylane-lightning
+
+.. image:: https://img.shields.io/codefactor/grade/github/PennyLaneAI/pennylane-lightning/master?logo=codefactor&style=flat-square
+    :alt: CodeFactor Grade
+    :target: https://www.codefactor.io/repository/github/pennylaneai/pennylane-lightning
+
+.. image:: https://img.shields.io/readthedocs/pennylane-lightning.svg?logo=read-the-docs&style=flat-square
+    :alt: Read the Docs
+    :target: https://pennylane-lightning.readthedocs.io
+
+.. image:: https://img.shields.io/pypi/v/PennyLane-Lightning.svg?style=flat-square
+    :alt: PyPI
+    :target: https://pypi.org/project/PennyLane-Lightning
+
+.. image:: https://img.shields.io/pypi/pyversions/PennyLane-Lightning.svg?style=flat-square
+    :alt: PyPI - Python Version
+    :target: https://pypi.org/project/PennyLane-Lightning
+
```

Comment:
![image](https://user-images.githubusercontent.com/49409390/90151584-dc39fa80-dd54-11ea-8d2b-953d5f1d27f5.png)

- Hopefully the build one will work once merged. Although this is the one I'm least sure about why it's working :thinking: 
- Coverage should come up once we merge this into master and upload a report
- Code quality should be improved in this PR, the low quality was due to one test file which is now improved
- The pypi and python shields should update when we release

Reply:
Update: the first one might be fixed with 8951a729a9913f13208517cf60a09389bfd65b2b